exegol/model/ContainerConfig.py

```diff
@@ -963,8 +963,6 @@ def addVolume(self,
                         logger.critical(msg)
                     raise CancelOperation(msg)
                 if path_match.startswith("/etc/"):
```

Comment:
Change to `if path_match.startswith("/etc/") and EnvInfo.isDockerDesktop():`

Reply:
I make the change :)